### PR TITLE
Fix 984

### PR DESCRIPTION
**Pull request by eroux**

### CODE REVIEW

**Comment by henryso** [comment]
I made the change you requested.  I think it does what it's supposed to, but I haven't made any TeX changes, so obviously the tests break.

File: tex/gregoriotex-main.tex [hunk 1]
```diff
@@ -1644,15 +1644,18 @@
 \newif\ifgre@in@euouae
 \gre@in@euouaefalse
 
+\newif\ifgre@firstin@euouae
+\gre@in@euouaefalse
```
Comment (rpspringuel) [inline tex/gregoriotex-main.tex]: As is this line is a repeat of 1645.  I think you meant `\gre@firstin@euouaefalse`

Comment (eroux) [inline tex/gregoriotex-main.tex]: thanks, addressed

**Comment by eroux** [comment]
should be ok now

**Comment by rpspringuel** [comment]
There are merge conflicts now, but other than that this looks good.

**Comment by eroux** [comment]
should be ok now

**Pull request merged**